http_service/bugbug_http/models.py

```diff
@@ -84,6 +80,8 @@ def preload_models():
 def classify_bug(
     model_name, bug_ids, bugzilla_token, expiration=DEFAULT_EXPIRATION_TTL
 ):
+    from bugbug_http.app import get_result_key
```

Comment:
Is it a problem if we add this as a top-level import?

Reply:
Yeah, it's a circular import :/

This could definitely be implemented better.. maybe when we create a `JobInfo` class it can move to a utility file or something.

Reply:
OK, let's file a follow-up so we don't forget